spec/model/user_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'User model' do
+    before(:each) do
+      @user = User.new(name: 'Test User', email: 'test@test.com', password: 'password')
+    end
+
+    it 'is valid with a name, email, and password' do
+      expect(@user).to be_valid
+    end
+
+    it 'is not valid without a name' do
+      @user.name = nil
+      expect(@user).to_not be_valid
+    end
+
+    it 'is not valid without an email' do
+      @user.email = nil
+      expect(@user).to_not be_valid
+    end
+
+    it 'is not valid without a password' do
+      @user.password = nil
+      expect(@user).to_not be_valid
+    end
+
```

Comment:
- [ ] These two tests are failing, Please make sure all tests pass without any errors.

![image](https://github.com/anthonymr/buddget/assets/83803414/8e41580e-5acc-4875-939e-2e2f99f82d86)

Reply:
Thaks, ill try running the test locally again.